data/remnant missions.txt

```diff
@@ -493,5 +592,771 @@ mission "Remnant: Technology Available"
+					goto HeavyLasers
+				`	"I'd rather not share that."`
+					decline
+			label HeavyLasers
+			`	"Really? Our records indicate that humanity had basic laser weapons when we left, but nothing that would be considered useful modern weapons." She pauses to look down at her scanner. "And yet our scans indicate that they are capable of significant power output."`
+			`	Based on these scans, we may well be able to learn quite a bit from these weapons. Would you be able to acquire a pair of them for us? We could offer you very good compensation for doing so."`
+			choice
+				`	"Sure, I would be glad to accept that deal."`
+				`	"Sorry, I'm really not interested in helping Remnant research."`
+					decline
+
+			`	After a bit of haggling, she agrees to pay you one and a half million credits in exchange for a cargo of two "Heavy Lasers."`
+				accept
+	on complete
+		outfit "Heavy Laser" -2
+		payment 1500000
+		conversation
+			`As the <ship> descends towards its assigned landing pad, you notice the weapons researcher mounting some kind of weapon on a platform aimed down a firing range. Jumping down from the stand, she pushes a button and a flash of light erupts from the muzzle, and the target downrange is cut in half. She flips a leaver and raises the barrel, and people rush out onto the range with tools and devices to examine the destruction.`
+			`	A freight platform is already waiting at the platform when you land, and about the time you get the cargo loaded the researcher shows up. She's dusty and looks a bit tired, but appears pleased with the results of her experiments. "Captain <first>, I see you have the lasers as we agreed." You aren't sure if Remnant rub their hands in eagerness, but she looks ready to do so.`
+			`	She taps her commlink, and moments later another Remnant appears from the scaffolding around a nearby Starling. The researcher straightens up and switchs to a more stately melody. "Captain <last>, this is Prefect Tali. She is our most experienced engineer for dealing with alien tech."`
+			`	"Greetings Captain. We appreciate your help in acquiring these samples. It looks like we are far behind the rest of humanity in some respects. We have a lot of work to do if we are to catch up with them."`
+
+
+
+mission "Remnant: Catalytic Ramscoop"
+	name "Retrieve Ramscoops for the Remnant"
+	description "An engineer on <planet>, a Remnant world, has offered to pay you one million credits in exchange for obtaining two Catalytic Ramscoops."
+	minor
+	source Viminal
+	to offer
+		has "license: Remnant"
+		random < 50
+	on offer
+		require "Catalytic Ramscoop"
+		conversation
+			`As you are walking through the spaceport, a woman comes up to you and introduces herself as an engineer at one of the Remnant development labs. She pauses awkwardly before beginning to chant "As your ship is the only one we have seen in generations that has outfits from ancestral space, we are intrigued by it. Could we take a closer look?"`
```

Comment:
I like "generations" better than "decades," because everything points towards the Remnant having been isolated for centuries from the rest of the humans, not just a few decades since their last update. Otherwise they'd probably have laser-based tech, and they'd know about the end of the war, and all sorts of other details.

Reply:
Changed the Void Sprite 1 wording to match.

Reply:
>I like "generations" better than "decades," because everything points towards the Remnant having been isolated for centuries from the rest of the humans, not just a few decades since their last update.

How long the Remnant have been gone from the rest of humanity and how long it's been since their last visitor are two totally different things.

Reply:
OK, I can revert that. I'm not sure it makes as much sense, though. If they'd had someone like the player just a decade or two past, then they'd already have updated history and everything.

Reply:
Actually, I just realized a small detail: Here he states "As your ship is the only one we have seen in generations that has weapons from human space..."

We know nothing about the previous ship. It is entirely possible, even likely, that it was a scoutship that didn't have any weapons on it, or perhaps just had some generic blasters. the Player's need for a tough ship to advance plot aside, it is much more likely to be a scoutship that reaches this far into the Ember Waste. No-one aside from the player is likely to waste that much crew wages to fly a warship around a chunk of space that has nothing in it.

And it is highly unlikely that such a scoutship would be carrying advanced beam weaponry.

Reply:
As a side note, I changed the catalyst one where it said generations to now refer to "many decades" like the VS1 mission does.